recovery/translation_nl.ts

```diff
@@ -362,22 +362,22 @@ Hierbij worden alle bestaande bestanden verwijderd.</translation>
     <message>
         <location filename="mainwindow.cpp" line="117"/>
         <source>Waiting for SD card (settings partition)</source>
-        <translation type="unfinished"></translation>
+        <translation>Wachten op SD kaart (settings partitie)</translation>
     </message>
     <message>
         <location filename="mainwindow.cpp" line="130"/>
         <source>Mounting settings partition</source>
-        <translation type="unfinished"></translation>
+        <translation>Mounten van de settings partitie</translation>
     </message>
     <message>
         <location filename="mainwindow.cpp" line="187"/>
         <source>Please wait while NOOBS initialises</source>
-        <translation>Even geduld</translation>
+        <translation>Even geduld terwijl NOOBS opstart</translation>
     </message>
     <message>
         <location filename="mainwindow.cpp" line="565"/>
         <source>Warning: incompatible Operating System(s) detected. The following OSes aren&apos;t supported on this revision of Raspberry Pi and may fail to boot or function correctly:</source>
-        <translation type="unfinished"></translation>
+        <translation>incompatibele OS gedetecteerd, the volgende OS&apos;s zijn niet ondersteund in deze versie van Raspberry Pi en kunnen mogelijk niet opstarten of correct functioneren</translation>
```

Comment:
the -> de
Verder wordt elders in het vertalingsbestand besturingssysteem i.p.v. OS gebruikt.